sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/main/java/com/azure/spring/sample/eventhubs/binder/EventProducerController.java

```diff
@@ -26,13 +26,13 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(EventHubBinderApplication.class);
 
     @Autowired
-    private EmitterProcessor<Message<String>> emitterProcessor;
+    private Sinks.Many<Message<String>> many;
 
     @PostMapping("/messages")
     public ResponseEntity<String> sendMessage(@RequestParam String message) {
-        LOGGER.info("Going to add message {} to emitter", message);
-        emitterProcessor.onNext(MessageBuilder.withPayload(message).build());
-        return ResponseEntity.ok("Sent!");
+        LOGGER.info("Going to add message {} to Sinks.Many.", message);
+        many.emitNext(MessageBuilder.withPayload(message).build(), Sinks.EmitFailureHandler.FAIL_FAST);
```

Comment:
could you explain more on this method?

Reply:
You can some code snippet in this post https://spring.io/blog/2020/07/10/first-milestone-of-reactor-2020-0-codename-europium